controllers/performanceprofile_controller.go

```diff
@@ -212,10 +170,10 @@ func (r *ReconcilePerformanceProfile) Reconcile(request reconcile.Request) (reco
 		// delete components
 		if err := r.deleteComponents(instance); err != nil {
 			klog.Errorf("failed to delete components: %v", err)
-			r.recorder.Eventf(instance, corev1.EventTypeWarning, "Deletion failed", "Failed to delete components: %v", err)
+			r.Recorder.Eventf(instance, corev1.EventTypeWarning, "Deletion failed", "Failed to delete components: %v", err)
```

Comment:
why is this public now? same for scheme, assetDir.

Reply:
This is how how the fields are generated by kubebuilder.

Reply:
Disappointing, but we can try to address in a followup OR